cmd/grpcurl/.goreleaser.yml

```diff
@@ -0,0 +1,22 @@
+builds:
+  - binary: grpcurl
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - 386
+
+archive:
+  wrap_in_directory: true
+  format: tar.gz
+  format_overrides:
+    - goos: windows
+      format: zip
+  replacements:
+    amd64: x64
+    386: x86
+    darwin: macos
+  files:
+    - LICENSE
```

Comment:
When I tried to run this locally, this failed because the `LICENSE` file is not in this directory. However, when I tried editing this to point to `../../LICENSE`, it generated a gnarly entry in the resulting TGZ that had `../LICENSE` as the path (e.g. it created a directory entry in the archive for `..`, which seems bad).

I have not yet tried to read through `goreleaser` docs to see how to fix this situation, but I think we need this configuration to run and produce a good archive before we merge this PR.

I also noticed that the TGZ has a directory inside named `grpcurl_<version>_<platform>_<arch>`, which seems quite verbose. Many binary distributions I've seen (IIRC) just have a folder named `grpcurl` (without the version, platform, and architecture). Is there a "best practice" that I don't know about for putting all of this information into the resulting directory after unzipping a TGZ?

Reply:
Ah yes. I originally had the `.goreleaser.yml` in the root, then moved it to the subdirectory. I can change the config so there's no nested directory. Some binaries I've seen don't both with the LICENSE, and they just have the binary without the directory. The easiest way to install these then is:

```
curl https://... | tar xz -C /usr/local/bin
```

Reply:
Adding the license can be nice if someone redistributes the binary, but practically for all of the users just downloading it to install immediately, it's an annoyance and it makes the install instructions slightly longer to extract only the binary from the tar.

Reply:
For now, I think we should keep the license in the distribution. I think it's simple enough to extract the binary into `/usr/local/bin`, for people that want to install it there.

In the future, I may look into adding a homebrew recipe, which will simplify it further for OS X users, and make it even easier to auto-install into that location.